### PR TITLE
Tailored Import - WIP feature

### PR DESCRIPTION
**Pull request by felix-couprie**
Feature is WIP

### CODE REVIEW

File: content/md/imports-exports/tailored-import.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ Thanks to Tailored Import, you can configure all your needs (mapping, operations
 To learn more about Tailored Import and how to use this feature, please watch our training video [Tailored Imports](https://akademy.akeneo.com/tailored-imports).
 :::
 
+:::info
+Tailored Import is a feature which will be updated all along the year.
```
Comment (ValentinMumble) [inline content/md/imports-exports/tailored-import.md:28]:
```suggestion
Tailored Imports is a feature which will be updated all along the year.
```

File: content/md/imports-exports/tailored-import.md [hunk 2]
```diff
@@ -126,7 +132,13 @@ Limitations have been set for each uploaded file:
 
 # Discover operations
 
-You can use operations to transform the data in order to make the import process easier. Depending on the targets that you define, here are the different types of operations that you can add:
+You can use operations to transform the data in order to make the import process easier.
+
+:::info
+New operations will come all during the year to help you save time importing your product information. Do not hesitate to contact your Customer Success Manager if you want to help us by communicating your use case & needs.
```
Comment (ValentinMumble) [inline content/md/imports-exports/tailored-import.md:138]:
```suggestion
New operations will come along the year to help you save time importing your product information. Do not hesitate to contact your Customer Success Manager if you want to help us by communicating your use case & needs.
```

File: content/md/imports-exports/tailored-import.md [hunk 3]
```diff
@@ -140,7 +152,7 @@ Replace the values by your own (only available for simple select, multi select<!
 
 ## Split
 
-If you want to import different values that are located into the same cell of your spreadsheet, you can use this operation to split the values thanks to a separator: comma (,), semicolon (;), pipe (|).
+If you want to import different values that are located into the same cell of your spreadsheet, you can use this operation to split the values thanks to a separator: comma (,), semicolon (; ), pipe (|).
```
Comment (ValentinMumble) [inline content/md/imports-exports/tailored-import.md:155]:
```suggestion
If you want to import different values that are located into the same cell of your spreadsheet, you can use this operation to split the values thanks to a separator: comma (,), semicolon (;), pipe (|).
```

**Review by ValentinMumble — APPROVED** [review]

File: content/md/imports-exports/tailored-import.md [hunk 4]
```diff
@@ -24,6 +24,12 @@ Thanks to Tailored Import, you can configure all your needs (mapping, operations
 To learn more about Tailored Import and how to use this feature, please watch our training video [Tailored Imports](https://akademy.akeneo.com/tailored-imports).
 :::
 
+:::info
+Tailored Import is a feature which will be updated all along the year.
+So far, the following attribute types aren't available to be mapped: `Price`, `Reference entity multi-select`, `Reference entity single-select`, `Asset collection`, `Image`, `Table`, `File` & `Image`.
+So far, the following system properties aren't available to be mapped: `Parent`, `Family variant`, `Groups` & `Associations`
```
Comment (StevenVAIDIE) [inline content/md/imports-exports/tailored-import.md:30]: Family variant is not really a product property so maybe here
```suggestion
So far, the following system properties aren't available to be mapped: `Parent`, `Groups` & `Associations`
```

Comment (StevenVAIDIE) [reply content/md/imports-exports/tailored-import.md:30]: Should we note that only XLSX product is available for now ?

**Pull request merged**